src/main/java/uk/ac/ebi/eva/pipeline/Application.java

```diff
@@ -34,6 +34,7 @@
 public class Application {
 
     private static final Logger log = LoggerFactory.getLogger(Application.class);
+    public static final String VARIANT_ANNOTATION_MONGO_PROFILE = "variant-annotation-mongo";
```

Comment:
Yeah, I didnt' dream it. http://docs.spring.io/autorepo/docs/spring-boot/current/reference/html/boot-features-profiles.html#boot-features-adding-active-profiles